docs/usage.rst

```diff
@@ -36,9 +36,10 @@ Examples:
 Django’s testing client
 -----------------------
 
-Attached to the context is an instance of TestCase.  You can access it
-via ``context.test``.  This means you can do things like use Django’s
-testing client.
+Internally, Django's TestCase or LiveServerTestCase is used to maintain database
+transactions and the live test server.
```

Comment:
Sure. But maybe it's more a "test live server" (i.e. a live server / runserser in the test run) rather than a "live test server" (a test server / a host for manual testing that behaves like a production aka "live" instance).

I know this discussion sounds ridiculous, but I'm sure it's not. The more self-explanatory it is, the less room for confusion there is, the better.